dart/lib/src/hub.dart

```diff
@@ -418,6 +419,7 @@ class Hub {
     Duration? autoFinishAfter,
     bool? trimEnd,
     OnTransactionFinish? onFinish,
+    String? origin,
```

Comment:
IIRC iOS added the `origin` as part of the `SentryTransactionContext` instead.

Reply:
Ah ok, we also have `origin` as part of `SentryTransactionContext` through inheritance from `SentrySpanContext` class. I will remove the extra param in the hub method.

Reply:
Updated the comment.

Reply:
No need to change it actually, we can just use the ctor param.